Xamarin.Essentials/TextToSpeech/TextToSpeech.android.cs

```diff
@@ -88,11 +84,16 @@ Task<bool> Initialize()
             return tcsInitialize.Task;
         }
 
-        public new void Dispose()
+        protected override void Dispose(bool disposing)
         {
-            tts?.Stop();
-            tts?.Shutdown();
-            tts = null;
+            if (disposing)
+            {
+                tts?.Stop();
+                tts?.Shutdown();
+                tts = null;
+            }
+
+            base.Dispose(disposing);
```

Comment:
Whaaa?! Just override the correct method, and not create a new method that will never be called.